spec/models/user_spec.rb

```diff
@@ -0,0 +1,37 @@
+# == Schema Information
+#
+# Table name: users
+#
+#  id                         :bigint           not null, primary key
+#  email                      :string(255)      not null
+#  password_hash              :string(255)
+#  token                      :uuid
+#  first_name                 :string(255)
+#  last_name                  :string(255)
+#  phone_number               :string(255)
+#  inserted_at                :datetime         not null
+#  updated_at                 :datetime         not null
+#  email_verification_token   :string(255)
+#  email_verified_at          :datetime
+#  role                       :string(255)      default("user"), not null
+#  password_reset_token       :uuid
+#  password_reset_expires_at  :datetime
+#  finalized                  :boolean          default(TRUE), not null
+#  avatar_key                 :uuid
+#  avatar_extension           :string(255)
+#  display                    :boolean          default(TRUE), not null
+#  terms_of_use               :datetime
+#  privacy_guidelines         :datetime
+#  agency_id                  :bigint
+#  last_active                :datetime
+#  status                     :string(255)
+#  active_session             :boolean          default(FALSE)
+#  renewal_request            :string(255)
+#  jwt_token                  :text
+#  recertification_expired_at :datetime
+#
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
let's add some basic specs to start out with, like maybe test it validates the presence of email and active_session is false by default